### PR TITLE
feat: add type annotations to jina drivers

### PR DESCRIPTION
**Pull request by rameshwara**
feat: add type annotations to jina drivers queryset

### CODE REVIEW

File: jina/drivers/__init__.py [hunk 1]
```diff
@@ -265,7 +265,7 @@ def _apply_all(self, docs: Iterable['jina_pb2.Document'], context_doc: 'jina_pb2
     def __call__(self, *args, **kwargs):
         self._traverse_apply(self.req.docs, *args, **kwargs)
 
-    def _traverse_apply(self, docs, *args, **kwargs):
+    def _traverse_apply(self, docs, *args, **kwargs) -> None:
```
Comment (JoanFM) [inline jina/drivers/__init__.py:268]: docs is an Iterable[jina_pb2.Document]

File: jina/drivers/__init__.py [hunk 2]
```diff
@@ -173,7 +173,7 @@ def __call__(self, *args, **kwargs) -> None:
         raise NotImplementedError
 
     @staticmethod
-    def _dump_instance_to_yaml(data):
+    def _dump_instance_to_yaml(data) -> Dict:
```
Comment (JoanFM) [inline jina/drivers/__init__.py:176]: Is there any hint on the type of data? And the actual type of Dict?

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by rameshwara** [comment]
Is there a reason why this is happening?

image

**Review by JoanFM — COMMENTED** [review]
Please keep a PR per package at least.

**Comment by rameshwara** [comment]
> Please keep a PR per package at least.

Done, have removed the additions in executors.

**Review by JoanFM — APPROVED** [review]

**Pull request merged**